improver/cli/subset_refrun_data.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python
```

Comment:
Although I understand why this CLI has this name, would it be better for it to be called something like `spatial_subset.py`?

Reply:
Yes, probably so.  Let me have a think.

Reply:
Done.